index.html

```diff
@@ -163,16 +163,37 @@ <h2>About me</h2>
                 </div>
             </section>
 
-             <div>
+            <section class="form-section">
+             <form action="https://formspree.io/f/mjvzwlqy" method="post">
                 <h2 class="touch">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
-                <br>
```

Comment:
- [ ] Kindly make sure you use the correct `fonts (size && weight)`, `line height`, `color` on the **"I'm always interested in hearing..."** text like the one used on the Figma design to stick to the expected design.

-- [ **OPTIONAL** ] **RECOMMENDED:** For the best semantics tag, I think it would be better if you use a `<p>` tag on the above description text instead of a `<h2>` tag as it is a `paragraph` and not a `title`.